### PR TITLE
BROKERS: Select Event By Id

### LINKED ISSUE #256: BROKERS: Select Event By Id

### PR DESCRIPTION
**Pull request by Hassaero110**
Closes #256

### CODE REVIEW

**Comment by Hassaero110** [comment]
Closes #256

File: Taarafo.Core/Brokers/Storages/StorageBroker.Events.cs [hunk 1]
```diff
@@ -35,5 +36,13 @@ public IQueryable<Event> SelectAllEvents()
 
             return broker.Events;
         }
+
+        public async ValueTask<Event> SelectEventByIdAsync(Guid eventID)
```
Comment (cjdutoit) [inline Taarafo.Core/Brokers/Storages/StorageBroker.Events.cs:40]: Change to eventId pls

File: Taarafo.Core/Brokers/Storages/StorageBroker.Events.cs [hunk 2]
```diff
@@ -35,5 +36,13 @@ public IQueryable<Event> SelectAllEvents()
...
+        {
+            using var broker =
+                new StorageBroker(this.configuration);
+
+            return await broker.Events.FindAsync(eventID);
```
Comment (cjdutoit) [inline Taarafo.Core/Brokers/Storages/StorageBroker.Events.cs:45]: Chnage to eventId

**Review by cjdutoit — CHANGES_REQUESTED** [review]
Please change the two items

**Review by cjdutoit — APPROVED** [review]

**Pull request merged**